.github/workflows/ci.yml

```diff
@@ -23,10 +23,8 @@ jobs:
         os: [ubuntu-latest]   # macOS-latest, windows-latest
         cabal: ["3.8"]
         ghc:
-          - "8.6.5"
           - "8.8.4"
           - "8.10.7"
-          - "9.0.2"
```

Comment:
I'm a bit fuzzy on the details but as I recall, the versions of transformers (or mtl) required to get `RWS.CPS` forces a version of `happy` that has incorrect bounds and doesn't compile. The easiest route was to disable those specific ghc versions, though I agree it's a bit unsatisfactory. If you know how to test bound resolution w/ ghc versions more easily (as opposed to updating the ci file and waiting for the github actions to fail), I'd be interested.

I think this is related: https://github.com/haskell/happy/issues/236

Reply:
Unfortunately I don't know of a way to test bounds :( 

Are you sure the CI builds currently fail? If so, can you keep the versions commented out instead of completely removing them + add a small explanatory comment that describes the problem (i.e. bounds do not work for RWS.CPS <~ happy).

Reply:
This is possibly also related to the [cabal.project.mtl23](https://github.com/agda/agda/blob/master/cabal.project.local.mtl23) unreleased version of `happy` in the Agda repository.

Reply:
I've added a new issue for that, so that this PR gets unblocked.